### PR TITLE
Bump Play to version 2.6

### PR DESCRIPTION
**Pull request by kelvin-chappell**

### CODE REVIEW

**Comment by kelvin-chappell** [comment]
[Upgrade Campaign Central to Play 2.6](https://trello.com/c/XTJ7Ecss/123-upgrade-campaign-central-to-play-26)

**Comment by LATaylor-guardian** [comment]
@kelvin-chappell looks okay to me but I'll default to @philwills given he done this for Riff Raff recently.

File: app/di.scala [hunk 1]
```diff
@@ -2,19 +2,27 @@ import controllers._
...
-class AppComponents(context: Context) extends BuiltInComponentsFromContext(context) with AhcWSComponents {
+class AppComponents(context: Context)
+  extends BuiltInComponentsFromContext(context)
+    with AhcWSComponents
+    with AssetsComponents
+    with HttpFiltersComponents {
+
+  // disabling some default filters until value is clear in this context
+  override def httpFilters: Seq[EssentialFilter] =
+    super.httpFilters.filterNot(filter => filter == csrfFilter || filter == allowedHostsFilter)
```
Comment (philwills) [inline app/di.scala:25]: If I'm reading this right, this disables the CSRFFilter. That seems like a bad idea to me unless there's a really compelling reason.

**Review by philwills — CHANGES_REQUESTED** [review]
With the exception of the filters, this looks reasonable.

File: app/di.scala (hunk 1 above)
Comment (kelvin-chappell) [reply app/di.scala:25]: My thinking is that the app will be no less secure than it is now with these filters disabled, it's clear that they are disabled, and I wanted to make sure that I haven't broken anything before enabling these filters one by one.  The CSRF filter looks like there will be quite a few changes involved.

**Review by philwills — APPROVED** [review]

File: app/di.scala (hunk 1 above)
Comment (philwills) [reply app/di.scala:25]: That makes sense, I had thought it was on by default in 2.5.

**Pull request merged**